### ISSUE TITLE
updating the pretrained network as well

### ISSUE THREAD

**Issue by baldmonkeey** [amasky/deep-learning-with-chainer#1]
Hi- Firstly This is not an issues, just a question.  I would like to allow the pre-trained network to update along with the along with the fc8ft. Would it suffice if I just comment out the model.caffe portion (or do I still have to explicitly specify the same learning rate for the pretrained network
thank!

**Comment by amasky** [comment]
Hi. This code actually updates all the layers of the model. 
Because Chainer seems not to accept different learning rates at different layers unlike Caffe, I set the 2 different learning rates to the 2 optimizers. The one is the pre-trained network "model.caffe", the other one is the added last layer "model.fine". Once we call "loss.backward()", then we get the grads of all the layers.

If you don't need to update the pre-trained layers, you should delete the "opt_caffe.update()
" or the all "opt_caffe*" lines. Also you can set the "volatile" option of "chainer.Variable", then the history of computations will not be held and "loss.backward()" computes only the last layers which flags are "volatile='off'" like this.

``` py
if train: x.volatile = 'on'
h = F.relu(self.caffe.conv1(x))
# forward of pre-trained net
if train: h.volatile = 'off'
h = self.fine.fc8ft(h)
return h
```

**Comment by baldmonkeey** [comment]
thanks!
